sdk/storage/azure-storage-blob/azure/storage/blob/_blob_client.py

```diff
@@ -1023,6 +1023,10 @@ def query_blob(self, query_expression, **kwargs):
             treat the blob data as CSV data formatted in the default dialect. This can be overridden with
             a custom DelimitedTextDialect, or DelimitedJsonDialect or "ParquetDialect" (passed as a string or enum).
             These dialects can be passed through their respective classes, the QuickQueryDialect enum or as a string
+
+            .. note::
+            "ParquetDialect" is in preview, so some features may not work as intended.
```

Comment:
This will render like this:
<img width="567" alt="image" src="https://github.com/Azure/azure-sdk-for-python/assets/101599632/c667ecde-474e-46dd-aea2-82a3cf8c2db6">
Pretty slick 😄